sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/IntegrationTestUtilities/SynchronizationContextHelper.cs

```diff
@@ -17,7 +16,7 @@ private static void SetSynchronizationContext()
             //Note that XUnit sets the sync context and we want to make sure not to pollute that - this means that
             //we cannot use the Assert.<> functions which take a lamdba and do an await inside XUnit code because it 
             //will cause a deadlock...
-            DispatcherSynchronizationContext synchronizationContext = new DispatcherSynchronizationContext();
```

Comment:
This was actually intentional -- can you IFDEF this and make it this way in netfx, and take your change in netcore only (for now at least)?

Reply:
Ok, will do.